packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -28,19 +28,19 @@ OAuth 2.0 is a standard that apps use to provide client applications with access
 
 The OAuth 2.0 spec has four important roles:
 
-* The "authorization server" &mdash; The server that issues the access token. In this case Okta is the authorization server.
+* **authorization server**: The server that issues the access token. In this case Okta is the authorization server.
```

Comment:
```suggestion
* **authorization server**: The server that issues the access token (Okta is the authorization server)
```
Suggest to remove "In this case" because we haven't referenced what case we are talking about?

Reply:
As discussed over Slack will stick with the period.